src/NServiceBus.Core/Routing/RoutingFeature.cs

```diff
@@ -87,5 +96,35 @@ protected internal override void Setup(FeatureConfigurationContext context)
                 }
             }
         }
+
+        void EnableBestPracticeEnforcement(FeatureConfigurationContext context)
+        {
+            var validations = new Validations(context.Settings.Get<Conventions>());
+
+            context.Pipeline.Register(
```

Comment:
Now that routing checks best practises do we still need this? (Yes bus.Send("explicitAddress") would bypass but that API is meant to give users full control anyway)